README.md

```diff
@@ -124,6 +124,8 @@ If `CASC_JENKINS_CONFIG` points to a folder, the plugin will recursively travers
 :x: `/jenkins/.configs/casc_configs/..dir2/config.yaml`  
 </details>
 
+Configuration file sets traversed in folders cannot overwrite configuration values, this will create a conflict and raise a `ConfiguratorException`. Thus, the order of traversal does not matter to the final outcome.
```

Comment:
```suggestion
All configuration files that are discovered MUST be supplementary. They cannot overwrite each other's configuration values. This creates a conflict and raises a `ConfiguratorException`. Thus, the order of traversal does not matter to the final outcome.
```

Reply:
Feel free to use this alternative phrasing if it seems more clear.

Reply:
+1 for alternate version. Thanks!